### DISCUSSION TITLE
User-Friendly Naming for Objects in Scene?

### DISCUSSION THREAD

**Discussion by thegladscientist** [mozilla/hubs#3156] [General Questions & Discussion] [answered]
Is there an option to change the way that objects are displayed in a scene in the object browser?

To show off 3D models on mobile, users have to go through the object browser, and many of the files end up with 'reticulum.io...6ee3f' as their names which is quite hard for people to understand and feel welcomed by. 

Is there an option to display the filename instead of the link name? Showing something like 'Ancient Mummy 12BC.fbx' might be more user-friendly in some cases, if this is possible.

PS I'm just on regular Hubs, not Hubs Cloud

**Comment by robinkwilson** [comment, marked as answer]
This is a great feature request! Made one here: https://github.com/mozilla/hubs/issues/3300
